<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainUI</class>
 <widget class="QWidget" name="mainUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1260</width>
    <height>980</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="ball" native="true">
   <property name="geometry">
    <rect>
     <x>234</x>
     <y>110</y>
     <width>180</width>
     <height>180</height>
    </rect>
   </property>
   <widget class="QPushButton" name="ballButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>V</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="background" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>488</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="bgLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetMinimumSize</enum>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QScrollArea" name="chatBoxScrollArea">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>177</height>
       </size>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="chatBox">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>472</width>
         <height>221</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="keyInputWidget" native="true">
      <property name="minimumSize">
       <size>
        <width>488</width>
        <height>58</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>488</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>20</number>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>29</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QLineEdit" name="keyInput">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="run">
         <property name="minimumSize">
          <size>
           <width>45</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>45</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../../../../../home/pratik/.designer/resource/send icon.png</normaloff>../../../../../../home/pratik/.designer/resource/send icon.png</iconset>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="controlButtonPlaceholder" native="true">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>-10</y>
     <width>420</width>
     <height>420</height>
    </rect>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>controlButtonPlaceholder</zorder>
  <zorder>ball</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>keyInput</sender>
   <signal>returnPressed()</signal>
   <receiver>run</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>371</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
